poetry/installation/installer.py

```diff
@@ -38,6 +38,7 @@ def __init__(
         self._locker = locker
         self._pool = pool
 
+        self._install_root = True
```

Comment:
I don't think this is needed. The check only needs to be done in the `install` command we don't need to modify the installer.